corehq/form_processor/submission_post.py

```diff
@@ -163,9 +162,6 @@ def run(self):
                 if instance.xmlns == DEVICE_LOG_XMLNS:
                     try:
                         process_device_log(self.domain, instance)
-                    except ProtocolError:
-                        # if riak is down/struggling, the caller will catch and notify
-                        raise
```

Comment:
doesn't this still mean that the `notify_exception` below gets called, which calls sentry, which won't catch it because it's not a raised error? Or does `notify_exception` still end up passing in `exc_info` to `should_capture` properly below? I'm now guessing that that's the case, but I'll leave this comment to make sure that was the reasoning